docs/content/wiring.md

```diff
@@ -54,6 +54,8 @@ The syntax to pass a parameter to porter is the same for both regular and file p
 $ porter install --param mytar=./my.tar.gz
 ```
 
+See the [Parameters section of the Author Bundles doc](author-bundles#Parameters) for additional examples and configuration.
```

Comment:
Try out this instead. The link didn't work in the netlify preview for me
```suggestion
See the [Parameters section of the Author Bundles doc](/author-bundles#parameters) for additional examples and configuration.
```

Reply:
Thank you!